NitroxServer/GameLogic/Entities/EntityData.cs

```diff
@@ -11,7 +11,7 @@ namespace NitroxServer.GameLogic.Entities
     [ProtoContract]
     public class EntityData
     {        
-        [ProtoMember(1)]
```

Comment:
Is this change needed anymore now that NitroxId has been updated?

Reply:
I checked, it isn't so i reverted it